docs/.vitepress/vitepress/components/vp-demo.vue

```diff
@@ -108,6 +109,18 @@ const copyCode = async () => {
       <el-collapse-transition>
         <SourceCode v-show="sourceVisible" :source="source" />
       </el-collapse-transition>
+      <transition name="el-fade-in-linear">
+        <div
+          v-show="sourceVisible"
+          class="example-float-control"
+          @click="() => setSourceVisible()"
```

Comment:
```suggestion
          @click="setSourceVisible"
```

Reply:
> 

I got it wrong here, it should be () => setSourceVisible(false)

Reply:
Just use `setSourceVisible(false)`